de.peterge.combined.yml

```diff
@@ -0,0 +1,32 @@
+app-id: de.peterge.combined
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: "Combined!.x86_64"
+
+finish-args:
+  - --socket=x11
```

Comment:
you need `- --share=ipc` for X11. And I suspect you also want `- --device=dri` for openGL.